sechub-commons-model/src/main/java/com/mercedesbenz/sechub/commons/model/JSONConverter.java

```diff
@@ -31,39 +36,49 @@ public static JSONConverter get() {
         return INSTANCE;
     }
 
-    private ObjectMapper mapper;
+    private JsonMapper mapper;
 
     public JSONConverter() {
-        // https://github.com/FasterXML/jackson-core/wiki/JsonParser-Features
-        JsonFactory jsonFactory = new JsonFactory();
-        jsonFactory.enable(JsonParser.Feature.ALLOW_COMMENTS);
-        jsonFactory.enable(JsonParser.Feature.ALLOW_SINGLE_QUOTES);
-
-        mapper = new ObjectMapper(jsonFactory);
-        /*
-         * next line will write single element array as simple strings. There was an
-         * issue with this when serializing/deserializing SimpleMailMessage class from
-         * spring when only one "to" defined but was an array - jackson had problems see
-         * also: https://github.com/FasterXML/jackson-databind/issues/720 and
-         * https://stackoverflow.com/questions/39041496/how-to-enforce-accept-single-
-         * value-as-array-in-jacksons-deserialization-process
-         */
-        mapper.enable(DeserializationFeature.ACCEPT_SINGLE_VALUE_AS_ARRAY);
-
-        /*
-         * we accept enums also case insensitive - e.g Traffic light shall be accesible
-         * by "GREEN" but also "green"...
-         */
-        mapper.enable(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS);
-
-        // but we do NOT use SerializationFeature.WRITE_SINGLE_ELEM_ARRAYS_UNWRAPPED !
-        // reason: otherwise jackson does all single ones write as not being an array
-        // which comes up to problems agani
-        mapper.disable(SerializationFeature.WRITE_SINGLE_ELEM_ARRAYS_UNWRAPPED);
+        /* @formatter:off */
+        mapper = JsonMapper.builder().
+            enable(JsonParser.Feature.ALLOW_COMMENTS).
+            enable(JsonParser.Feature.ALLOW_SINGLE_QUOTES).
+            /*
+             * next line will write single element array as simple strings. There was an
+             * issue with this when serializing/deserializing SimpleMailMessage class from
+             * spring when only one "to" defined but was an array - jackson had problems see
+             * also: https://github.com/FasterXML/jackson-databind/issues/720 and
+             * https://stackoverflow.com/questions/39041496/how-to-enforce-accept-single-
+             * value-as-array-in-jacksons-deserialization-process
+             */
+            enable(DeserializationFeature.ACCEPT_SINGLE_VALUE_AS_ARRAY).
+
+            /* but we do NOT use SerializationFeature.WRITE_SINGLE_ELEM_ARRAYS_UNWRAPPED !
+             reason: otherwise jackson does all single ones write as not being an array
```

Comment:
I read this comment more than 3 times, but I still not understand it. Please try to improve the sentence.

In addition, please try to be consistent with your own style and add `*` in front of the line and in a single line the `*/`.

```
             reason: otherwise jackson does all single ones write as not being an array which comes up to problems again*/
```

Reply:
I will remove the comment - if there is anything unclear, people are able to read the javadoc description from jackson.